scss/bootstrap/components/civicase/_chat-bubble.scss

```diff
@@ -0,0 +1,135 @@
+.chat-bubble {
+  margin: 10px 0;
+  padding: 0;
+  
+  &.chat-user-left {
+    .chat-user {
+      float: left;
+    }
+
+    .chat-cont {
```

Comment:
I belive this class name should be improved as I mentioned in this comment 

https://github.com/civicrm/org.civicrm.shoreditch/pull/22#issuecomment-319340580